### ISSUE TITLE
Error occurred when using the pre trained models you provided

### ISSUE THREAD

**Issue by HJ-Xu** [RobinMagnet/SimplifiedFmapsLearning#2]
Thank you very much for uploading the complete code. However, when I attempted to reproduce your results, I encountered the following error:
![1727708783587](https://github.com/user-attachments/assets/bd50046b-dc37-4fd1-be96-cc506f38514d)
And, when I try to train the model, there are still errors, which I think may be related to the software environment. Can you disclose the software environment in which you run the code, including Python and pytorch versions, etc
![image](https://github.com/user-attachments/assets/bf49c2be-4c52-4f0f-971e-cc319e201f8f)

**Comment by louisehuang07** [comment]
Hello! I’ve encountered a similar issue. Could you please share if you’ve found a solution or have any advice on how to resolve it? Thank you in advance for your assistance.

**Comment by louisehuang07** [comment]
> Thank you very much for uploading the complete code. However, when I attempted to reproduce your results, I encountered the following error: ![1727708783587](https://private-user-images.githubusercontent.com/44128932/372142233-bd50046b-dc37-4fd1-be96-cc506f38514d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgwMjcwMDQsIm5iZiI6MTcyODAyNjcwNCwicGF0aCI6Ii80NDEyODkzMi8zNzIxNDIyMzMtYmQ1MDA0NmItZGMzNy00ZmQxLWJlOTYtY2M1MDZmMzg1MTRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA0VDA3MjUwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQxOTU1NGJkZmFmNjg3NGRkMDE5ODJlNjkyNDg0MDc0MWFiZTZkNjZlYWExNWEwMDVlZDRhZmQxYmIyNDQxODgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JYKNV-8Aj_tWq1ETmGXbrtpBG64kiEdv1TyaECX5MPU) And, when I try to train the model, there are still errors, which I think may be related to the software environment. Can you disclose the software environment in which you run the code, including Python and pytorch versions, etc ![image](https://private-user-images.githubusercontent.com/44128932/372143125-bf49c2be-4c52-4f0f-971e-cc319e201f8f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgwMjcwMDQsIm5iZiI6MTcyODAyNjcwNCwicGF0aCI6Ii80NDEyODkzMi8zNzIxNDMxMjUtYmY0OWMyYmUtNGM1Mi00ZjBmLTk3MWUtY2MzMTllMjAxZjhmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA0VDA3MjUwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIyNzFiODYwMGE0NDc0MTU1NTg3NjU5ODkzYzU1NTlhYjBjZjU5YTc5ZTY2Y2I0ODgxY2E5MWQ4YjA3ZTg2MGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JyHgJNz7eZ_BRGiI2k1tJEAh0s5B8aFDTsXxVRHiqf0)

Hello! I’ve encountered a similar issue. Could you please share if you’ve found a solution or have any advice on how to resolve it? Thank you in advance for your assistance.

**Comment by RobinMagnet** [comment]
Hi,

I'm very sorry to see this. It seems I get the same issue for checkpoints of runs that were launched on a specific cluster, which I hope I still have access to. It might just be an issue with downloading the checkpoint, I'll try to see what I can do asap.

I think all experiments were run on python 3.9, with torch 2.1.0 and CUDA 12.2. But the experiments for which the checkpoints are failing might have used slightly different packages, I need to see if I can find them.

I will check additional information on the environment and let you know when I'm sure about it.

**Comment by xiangdonglai** [comment]
It seems to be the case that when the dataset class creates `10_mesh_256k_0n_descnorm.pt` in `_prepare`, but tries to load `10_mesh_256k_0n.pt` in `__getitem__`. I am wondering whether `normalize_desc` is set to True for the experiments?

**Comment by HJ-Xu** [comment]
> > 非常感谢您上传完整的代码。但是，当我尝试重现您的结果时，我遇到了以下错误：![1727708783587](https://private-user-images.githubusercontent.com/44128932/372142233-bd50046b-dc37-4fd1-be96-cc506f38514d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgwMjcwMDQsIm5iZiI6MTcyODAyNjcwNCwicGF0aCI6Ii80NDEyODkzMi8zNzIxNDIyMzMtYmQ1MDA0NmItZGMzNy00ZmQxLWJlOTYtY2M1MDZmMzg1MTRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA0VDA3MjUwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQxOTU1NGJkZmFmNjg3NGRkMDE5ODJlNjkyNDg0MDc0MWFiZTZkNjZlYWExNWEwMDVlZDRhZmQxYmIyNDQxODgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JYKNV-8Aj_tWq1ETmGXbrtpBG64kiEdv1TyaECX5MPU)而且，当我尝试训练模型时，仍然存在错误，我认为这可能与软件环境有关。您能否透露运行代码的软件环境，包括 Python 和 pytorch 版本等![image](https://private-user-images.githubusercontent.com/44128932/372143125-bf49c2be-4c52-4f0f-971e-cc319e201f8f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgwMjcwMDQsIm5iZiI6MTcyODAyNjcwNCwicGF0aCI6Ii80NDEyODkzMi8zNzIxNDMxMjUtYmY0OWMyYmUtNGM1Mi00ZjBmLTk3MWUtY2MzMTllMjAxZjhmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA0VDA3MjUwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIyNzFiODYwMGE0NDc0MTU1NTg3NjU5ODkzYzU1NTlhYjBjZjU5YTc5ZTY2Y2I0ODgxY2E5MWQ4YjA3ZTg2MGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JyHgJNz7eZ_BRGiI2k1tJEAh0s5B8aFDTsXxVRHiqf0)
> 
> 你好！ 我遇到了类似的问题。您能否分享一下您是否找到了解决方案或对如何解决它有任何建议？提前感谢您的帮助。

hi
I'm sorry I couldn't find a solution, but I found that during the training process, only the file name of the data in the data processing file needs to be modified to train normally. Although this cannot fully reproduce the results reported in the paper, approximate accuracy can be obtained.

**Comment by RobinMagnet** [comment]
Hi,

I have investigated a bit, and it seems that the checkpoints were when I downloaded them. Sadly, I have lost all data related to this. I recently gained back access to the cluster and I am working on retraining everything again.

The training should be deterministic since everything is seeded, so I am surprised the results are not the same as the one in the paper. This is probably due to torch/cuda versions being different. In particular, the config files are provided are the one used to produce the checkpoints in the same folders.

Again I am very sorry about this, it's pretty embarrassing for me.
